### ISSUE TITLE
Xeno Industry does not appear to be able to modify the Clusterio storage.

### ISSUE THREAD

**Issue by Nihilhalla** [Rahjital/XenoIndustry#2]
So I've tested this as many ways as I could think to, but a remote install of KSP with the XenoIndustry mod doesn't seem to be able to modify the storage. For example, I made a rocket in KSP but it didn't remove the resources from the Clusterio server. Furthermore I wasn't able to add science packs from KSP. I enabled Debug mode in config.json but didn't see where to get a log file for you. Sorry if I'm prodding into a mod with no planned future development.

**Comment by Danielv123** [comment]
Did you apply your secret-api-token.txt to Xenoindustry in any way? It is required for clusterio since 3 months ago.

**Comment by Rainemakerx** [comment]
{
	"masterIP": "<IP here>",
	"masterPort": "8080",
	"masterAuthToken": "<Master Token from Secret-API-Text here>"
	"debug": "true",
}

**Comment by Nihilhalla** [comment]
I applied that to an added field 'masterAuthToken: ""'
quotation marks contain the actual token, apostrophes contain the exact field.

**Comment by Danielv123** [comment]
If you did that and correctly configured the masterIP and port as well I am afraid I can't help you. Will have to wait for @Rahjital

**Comment by Nihilhalla** [comment]
That's what I was figuring on. The mod receives information just fine, but can't alter counts from the cluster at all it seems. To confirm, the master port is the same as the master port configured in the clusterio config yes? I mean I would think it would have to be if I was able to get any information at all and not just a config error.

**Comment by Danielv123** [comment]
Yes, it is. And if its the same system the masterIP would be "localhost"

**Comment by TeacupAngel** [comment]
That it is very peculiar, can't say I encountered an issue like this before. Can you please upload your KSP.log file (usually found in the root KSP directory, next to the exectuable) and also both your XenoIndustry and Clusterio config files, through Pastebin or a similar service?

The mod is still in development, although it has been stalled rather badly on Unity's ancient version of .NET.

**Comment by Nihilhalla** [comment]
https://pastebin.com/JEfBu3G2 <XI Config.json
https://pastebin.com/d6uPdBf0 <Clusterio instance Config.json
[KSP.log](https://github.com/Rahjital/XenoIndustry/files/2131274/KSP.log)
I hope you don't mind, but I obscured most of the sensitive information to protect my privacy.

**Comment by TeacupAngel** [comment]
Finally got to take a good look at this. For some reason the sent to Clusterio requests fail, but the error message given by Unity is very much nondescriptive ("Generic/unknown HTTP error"). It could be that the new versions of Clusterio are no longer compatible, or some other connection issue. I'll see if I can reproduce this and hopefully resolve this as well.
